source/frontend/src/features/leases/detail/LeasePages/details/UpdateLeaseForm.tsx

```diff
@@ -18,27 +20,36 @@ export const UpdateLeaseForm: React.FunctionComponent<
   React.PropsWithChildren<IUpdateLeaseFormProps>
 > = ({ onSubmit, initialValues, formikRef }) => {
   return (
-    <Formik
-      validationSchema={AddLeaseYupSchema}
-      onSubmit={values => onSubmit(values)}
-      initialValues={initialValues ?? getDefaultFormLease()}
-      innerRef={formikRef}
-    >
-      {formikProps => (
-        <>
-          <Prompt
-            when={formikProps.dirty}
-            message="You have made changes on this form. Do you wish to leave without saving?"
-          />
-          <Styled.LeaseForm>
-            <LeaseDetailSubForm formikProps={formikProps}></LeaseDetailSubForm>
-            <AdministrationSubForm formikProps={formikProps}></AdministrationSubForm>
-            <ReferenceSubForm />
-          </Styled.LeaseForm>
-        </>
-      )}
-    </Formik>
+    <StyledFormWrapper>
+      <Formik<LeaseFormModel>
+        validationSchema={AddLeaseYupSchema}
+        onSubmit={values => onSubmit(values)}
+        initialValues={initialValues ?? getDefaultFormLease()}
+        innerRef={formikRef}
+      >
+        {formikProps => (
+          <>
+            <Prompt
+              when={formikProps.dirty}
+              message="You have made changes on this form. Do you wish to leave without saving?"
+            />
+            <Styled.LeaseForm>
+              <LeaseDetailSubForm formikProps={formikProps}></LeaseDetailSubForm>
+              <LeasePropertySelector formikProps={formikProps} />
```

Comment:
Hmm, okay, this should actually work with my changes without needing any additional fixes, would still prefer that you merge first so that I can rebase and confirm.